### PR TITLE
Fix crash when sshDetails is null

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

when `sshDetails` is `null`, the catalog drawer crash.

## 🎥 Demo

**Before**

image

**After**

Screenshot from 2024-02-20 13-40-14

### CODE REVIEW

File: i18n-scripts/set-english-defaults.js [hunk 1]
```diff
@@ -48,11 +49,14 @@ function updateFile(fileName) {
       switch (determineRule(keys[i])) {
         case 0:
           [originalKey] = keys[i].split('_other');
-          updatedFile[keys[i]] = `{{count}} ${pluralize(originalKey)}`;
+
+          wordWithNoCount = originalKey.replace('{{count}}', '').trim()
```
Comment (hstastna) [inline i18n-scripts/set-english-defaults.js:53]: Are these changes in this file intentional? Feels like those don't belong to this PR...

Comment (upalatucci) [reply i18n-scripts/set-english-defaults.js:53]: No i pushed in the wrong branch

Comment (upalatucci) [reply i18n-scripts/set-english-defaults.js:53]: fixing

**Review by hstastna — APPROVED** [review]

**Comment by avivtur** [comment]
/hold
Holding until upgrade to PF5 is completed
